### ISSUE TITLE
Use chameleon container

### ISSUE THREAD

**Issue by rmpestano** [manorrock/jakartaee-examples#2]
[chameleon container](https://github.com/arquillian/arquillian-container-chameleon) is an arquillian proxy container that makes it easier switching ad testing against different arquillian containers.

**Comment by manorrock** [comment]
There is no way to avoid the use of arquillian.xml?

**Comment by rmpestano** [comment]
Don't know, @lordofthejars, @bartoszmajsak?

**Comment by manorrock** [comment]
And did the change go into chameleon?

**Comment by rmpestano** [comment]
Yes, it was merged but not released yet. As almost all pending PRs have been merged in chameleon last week I expect a release soon.

**Comment by lordofthejars** [comment]
We are having some problems for releasing it, but working on that.

El 8 sept. 2018 4:26 p. m., "Rafael M. Pestano" <notifications@github.com>
escribió:

Yes, it was merged but not released yet. As almost all pending PRs have
been merged in chameleon last week I expect a release soon.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/manorrock/jakartaee-examples/issues/2#issuecomment-419645825>,
or mute the thread

**Comment by bartoszmajsak** [comment]
`1.0.0.CR3` should land in Maven Central very soon.  Thanks for your contribution @rmpestano!

**Comment by rmpestano** [comment]
@manorrock I've just tested with chameleon CR3 and it's all green here:

![jakartaee-tests](https://user-images.githubusercontent.com/1592273/45268506-d5612980-b453-11e8-8ddb-7a5f1fb02f32.png)

**Comment by manorrock** [comment]
Sweet, is the PR ready to be merged?

**Comment by rmpestano** [comment]
Yeap, It is already merged with your repo.

**Comment by manorrock** [comment]
Looks like it is not passing the automated tests on Travis, see https://travis-ci.org/manorrock/jakartaee-examples/pull_requests

**Comment by rmpestano** [comment]
I'm not a desktop now but looks like Travis is not running against the latest commit. It is failing because it doesn't find arquillian-chameleon 1.0.0-SNAPSHOT but on my repo chameleon version is 1.0.0-CR3 ([see here](https://github.com/rmpestano/jakartaee-examples/blob/a25c35655dc351aa3fc050102b8a96520bf47c2d/pom.xml#L106))

**Comment by rmpestano** [comment]
On my git três the latest commit is `a25c` but Travis is building against `521c`

![screenshot_20180912-195539](https://user-images.githubusercontent.com/1592273/45457614-3a23ba80-b6c6-11e8-9b3e-47a2ad4876d6.png)

**Comment by manorrock** [comment]
Lets create a new PR and abandon the one that TravisCI seems to have an issue with.

**Comment by rmpestano** [comment]
Still failing trying to resolve chameleon SNAPSHOT, don't where it is comming from, can you try to `git clone https://github.com/rmpestano/jakartaee-examples` and build the project?

**Comment by manorrock** [comment]
It is coming from test-commons/pom.xml

**Comment by rmpestano** [comment]
:+1:  I knew Travis were right ;) (adjusted from mobile, I hope It is right now)

**Comment by rmpestano** [comment]
One small tip to make Travis faster is to cache maven repo and .arquillian folder [see example here](https://github.com/rmpestano/cdi-crud/blob/9b71cf616026ee2cdf9981cad3558d1cff8c9f83/.travis.yml#L4)

**Comment by manorrock** [comment]
Can I update the copyright / add the copyright on the new files to be the same as the rest of the Git repository?

**Comment by rmpestano** [comment]
As far as I remember it is just the JakartaEEServer annotation. Do I need to update @author on all tests because I modified them?

**Comment by manorrock** [comment]
No not needed, unless you want to add yourself to that list there ;)
